src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__description {
+    color: $main-text-color;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+  }
+
+  &__code {
+    color: $secundary-text-color;
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+    &-reviews {
+      color: $main-text-color;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+    }
```

Comment:
The font-size of the reviews is also set to 10px. As with the code, consider increasing the font-size to at least 12px to improve readability.